### PR TITLE
Get the name from node's obj attribute

### PR DESCRIPTION
**Pull request by vodik**

### CODE REVIEW

**Comment by vodik** [comment]
Bump?

File: interactive/plugin.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def gen_nodes(item, cache):
     chain = item.listchain()
     for node in chain:
         try:
-            name = node._obj.__name__
+            name = node.obj.__name__
```
Comment (tgoodlet) [inline interactive/plugin.py]: @vodik nice. Yeah only thing is do we need to pin **after** a particular `pytest` version now?

Comment (vodik) [inline interactive/plugin.py]: I don't think so, but I'll check tonight.

Comment (tgoodlet) [inline interactive/plugin.py]: @vodik did you?

**Comment by tgoodlet** [comment]
@vodik pretty sure #14 has something to do with this as well.

**Comment by tgoodlet** [comment]
Close in favor of #20. 
The solution here is incorrect and would only be non-`None` for test objects which are *instances*.

**Pull request closed without merging**